examples/5-nextjs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "scaffold-test",
+  "scripts": {
+    "install-mst-gql": "bash ../../scripts/install-mst-gql.sh",
+    "scaffold": "../../generator/mst-gql-scaffold.js --roots 'Todo' --excludes 'CacheControlScope,Query,Subscription' --outDir src/models/ --format ts http://localhost:4000/graphql",
+    "build": "tsc",
```

Comment:
should build also include next build? I'm not really an expert on next though, so you probably know that better than I do. I'm also missing a script to start the server?